config/application.rb

```diff
@@ -29,7 +29,6 @@ module VetsAPI
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
-    config.autoloader = :zeitwerk
```

Comment:
So removing this line enables zeitwerk?

Reply:
Yes, zeitwerk is the default for 6.0.  The `config.autoloder = :classic` was added to override defaults.